moped-editor/src/utils/humanReadableFileSize.js

```diff
@@ -16,8 +15,8 @@ export default (bytes, si=false, dp=1) => {
     }
 
     const units = si
-        ? ['kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
-        : ['KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB', 'ZiB', 'YiB'];
+        ? ['KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
+        : ['KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```

Comment:
I'm confused what's happening here now. Both conditions look identical.

Reply:
Ah, good call. Yea they are now identical. I've removed the two lists and we only have one now.